utils/rowgroup/rowgroup.h

```diff
@@ -1413,15 +1361,19 @@ class RowGroup : public messageqcpp::Serializeable
     // Used to map the projected column and rowgroup index
     std::vector<uint32_t> keys;
     std::vector<execplan::CalpontSystemCatalog::ColDataType> types;
-
-    // DECIMAL support.  For non-decimal fields, the values are 0.
+    // For string collation
+    std::vector<uint32_t> charsetNumbers;
+    std::vector<CHARSET_INFO*> charsets;
```

Comment:
Is it safe to assume CHARSET_INFO life cycle is bigger then that of the RowGroup?

Reply:
Yes. CHARSET_INFO is singleton life of the program